dkpro-core-opennlp-asl/pom.xml

```diff
@@ -124,6 +124,12 @@
       <artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-model-sentence-en-maxent</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+		<groupId>de.tudarmstadt.ukp.dkpro.core</groupId>
+		<artifactId>de.tudarmstadt.ukp.dkpro.core.opennlp-model-tagger-af-afribooms</artifactId>
+		<version>20181017.1</version>
```

Comment:
Instead of specifying the version in the dependency section, remove it here and add the dependency (without scope) also in the dependency management section. The latter is also necessary for model auto-loading to work. Mind that there may be two dependency sections in a POM - one for normal builds and one for full builds with all models. However, there is always only a single dependency management section which must not be in a profile.